rpc/core/pipe.go

```diff
@@ -89,6 +90,10 @@ func SetTxIndexer(indexer txindex.TxIndexer) {
 	txIndexer = indexer
 }
 
+func SetConsensusReactor(conR *consensus.ConsensusReactor) {
+	consensusReactor = conR
```

Comment:
Currently we're structured to only call this once from node/node.go at setup time. But we could certainly use comments/docs reflecting that fact!